### DISCUSSION TITLE
How does the pivotbar manage overflow?

### DISCUSSION THREAD

**Discussion by mikezimm** [n8design/htwoo#9] [Ideas] [answered]
Hi,
One of the biggest challenges I have with using Fluent UI in SPFx projects is lack of responsive design in the Pivot compnent.

Does this solve that issue?  I can't tell from this example since it only has 3 buttons in the example.

https://lab.n8d.studio/htwoo/htwoo-core/?p=molecules-pivotbar

Here's an example of my current webpart which allows for a rather lengthy set of pivot buttons.
Also complicating my implementation is that I also want to have a "farRight" set of buttons that float to the right which are not part of the pivot.

Right now, if I load this on an ipad, I can't even get to my farRight buttons :(

![image](https://user-images.githubusercontent.com/49648086/116645691-54c50380-a944-11eb-8be7-0e3da1213472.png)

Thanks!

**Comment by StfBauer** [comment]
Thank you @mikezimm for the suggestion. That is a really good point.

Can you do me a favour and add an issue. While it would be fairly easy to accomplish it is not yet supported.

It will absolutely a great add-on to implement.

**Comment by StfBauer** [comment, marked as answer]
For quite some time @mikezimm the support for[ pivot bars](https://lab.n8d.studio/htwoo/htwoo-core/?p=molecules-pivotbar-overflow) and [command bars](https://lab.n8d.studio/htwoo/htwoo-core/?p=molecules-cmdbar-overflow) have been added. 

UP next coming will be the support for facepile elements.
